miio/integrations/vacuum/roborock/vacuumcontainers.py

```diff
@@ -502,6 +555,11 @@ def area(self) -> float:
         """Total cleaned area."""
         return pretty_area(self.data["area"])
 
+    @property
+    def multi_map_id(self) -> int:
```

Comment:
```suggestion
    def current_map_id(self) -> int:
```
Maybe to make it clearer what it is?

Reply:
Yes I agree, I changed it